### PR TITLE
[15.0][MIG] project_parent_task_filter

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Migration to v15.

@ForgeFlow

### CODE REVIEW

File: project_parent_task_filter/models/project.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+class TaskSub(models.Model):
+    _inherit = "project.task"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super(TaskSub, self).create(vals_list)
+        for vals in vals_list:
+            if vals.get("parent_id", False):
```
Comment (ChrisOForgeFlow) [inline project_parent_task_filter/models/project.py]: It might be a good idea to add an init hook to ensure that the historical data remains consistent.

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]

**Review by ChrisOForgeFlow — APPROVED** [review]
Code Review +1

**Review by tarteo — APPROVED** [review]
Code LGTM, tested locally. Just one small non-blocking request

File: project_parent_task_filter/models/project.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
+from odoo import api, models
```
Comment (tarteo) [inline project_parent_task_filter/models/project.py]: Can you rename this file to `project_task.py`

**Review by JoanMForgeFlow — APPROVED** [review]
LG!

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

File: project_parent_task_filter/models/project_task.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    self.env["project.task"].browse(vals.get("parent_id")).project_id.id
+                )
+        return res
+
+    def action_subtask(self):
+        action = {
+            "type": "ir.actions.act_window",
+            "name": "Subtasks of " + self.name,
+            "res_model": "project.task",
+            "view_mode": "tree,kanban,form,calendar,pivot,graph,activity",
```
Comment (emagdalenaC2i) [inline project_parent_task_filter/models/project_task.py]: Could you change this? 
```suggestion
            "view_mode": "kanban,tree,form,calendar,pivot,graph,activity",
```

**Review by Mantux11 — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
Functional review

**Pull request merged**